packages/ra-core/src/controller/list/useList.spec.tsx

```diff
@@ -21,70 +21,6 @@ const UseList = ({
 };
 
 describe('<useList />', () => {
-    it('should filter string data based on the filter props', () => {
```

Comment:
I moved these tests further down to group them in the `describe('filter')` group, I didn't remove or modify them